### DISCUSSION TITLE
Anybody successfully fine-tuning FastPitch

### DISCUSSION THREAD

**Discussion by helgadnestr** [NVIDIA-NeMo/NeMo#6418] [Q&A]
Hi everyone
Has anybody successfully fine-tuning FastPitch with small dataset (about 30 minutes duration)? Could you share examples generated records after trained, it's important for me to know that it's possible to get records sound like human after fine-tuning with small good dataset.
And another question, can I expect good quality do FastPitch fine-tuning with small dataset Male voice on pretrained model with Woman voice?

**Comment by oytunturk** [comment]
It won't sound "human-like" but, yes, adaptation works using 30 minutes or even less data. Even with 20+ hours of data and training from scratch, it won't sound as natural as human speech. It depends on the quality of the neural vocoder you are using on top of FastPitch.
